index.html

```diff
@@ -98,7 +98,7 @@ <h2>Profesional Art Printing Data</h2>
                             <h2>Profesional Art Printing Data</h2>
                             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
                                 the industry's standard</p>
-                            <div class="luang">
+                            <div class="technologies">
```

Comment:
Hey, @Regiss05 professional structure, but I think you may use section instead of div in sequence   😀